spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/instance/Instance.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2020-2022 Huawei Technologies Co., Ltd. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.huaweicloud.common.instance;
+
+
+import org.apache.servicecomb.service.center.client.model.MicroserviceInstance;
+import org.springframework.cloud.client.ServiceInstance;
+
+public interface Instance {
+  MicroserviceInstance getMicroserviceInstance();
```

Comment:
do not need this interface. Can add more operations to `MicroserviceInstanceService`

Reply:
Modified